cms/djangoapps/contentstore/views/transcript_settings.py

```diff
@@ -276,14 +264,13 @@ def transcript_delete_handler(request, course_key_string, edx_video_id, language
         language_code: transcript's language code.
 
     Returns
-        - A 404 if the corresponding feature flag is disabled or user does not have required permisions
+        - A 404 if the user does not have required permisions
         - A 200 if transcript is deleted without any error(s)
     """
     # Check whether the feature is available for this course.
     course_key = CourseKey.from_string(course_key_string)
-    video_transcripts_enabled = VideoTranscriptEnabledFlag.feature_enabled(course_key)
     # User needs to have studio write access for this course.
-    if not video_transcripts_enabled or not has_studio_write_access(request.user, course_key):
+    if not has_studio_write_access(request.user, course_key):
```

Comment:
why do we need to check write access here ?

Reply:
so that not everyone can delete the transcript